language/German/strings.po

```diff
@@ -4565,1083 +4626,1083 @@ msgstr ""
+msgstr "AVI aufzeichnen"
 
 #: src/emu/inpttype.ipp:891
 msgctxt "input-name"
 msgid "Toggle Cheat"
-msgstr ""
+msgstr "Cheat umschalten"
 
 #: src/emu/inpttype.ipp:892
 msgctxt "input-name"
 msgid "UI Up"
-msgstr ""
+msgstr "Benutzeroberfläche Hoch"
 
 #: src/emu/inpttype.ipp:893
 msgctxt "input-name"
 msgid "UI Down"
-msgstr ""
+msgstr "Benutzeroberfläche Runter"
 
 #: src/emu/inpttype.ipp:894
 msgctxt "input-name"
 msgid "UI Left"
-msgstr ""
+msgstr "Benutzeroberfläche Links"
 
 #: src/emu/inpttype.ipp:895
 msgctxt "input-name"
 msgid "UI Right"
-msgstr ""
+msgstr "Benutzeroberfläche Rechts"
 
 #: src/emu/inpttype.ipp:896
 msgctxt "input-name"
 msgid "UI Home"
-msgstr ""
+msgstr "Benutzeroberfläche Pos1"
 
 #: src/emu/inpttype.ipp:897
 msgctxt "input-name"
 msgid "UI End"
-msgstr ""
+msgstr "Benutzeroberfläche Ende"
 
 #: src/emu/inpttype.ipp:898
 msgctxt "input-name"
 msgid "UI Page Up"
-msgstr ""
+msgstr "Benutzeroberfläche Bild Auf"
 
 #: src/emu/inpttype.ipp:899
 msgctxt "input-name"
 msgid "UI Page Down"
-msgstr ""
+msgstr "Benutzeroberfläche Bild Ab"
 
 #: src/emu/inpttype.ipp:900
 msgctxt "input-name"
 msgid "UI Focus Next"
-msgstr ""
+msgstr "Benutzeroberfläche Nächster Fokus"
 
 #: src/emu/inpttype.ipp:901
 msgctxt "input-name"
 msgid "UI Focus Previous"
-msgstr ""
+msgstr "Benutzeroberfläche Voriger Fokus "
 
 #: src/emu/inpttype.ipp:902
 msgctxt "input-name"
 msgid "UI Select"
-msgstr ""
+msgstr "Benutzeroberfläche Auswählen"
 
 #: src/emu/inpttype.ipp:903
 msgctxt "input-name"
 msgid "UI Cancel"
-msgstr ""
+msgstr "Benutzeroberfläche Abbrechen"
 
 #: src/emu/inpttype.ipp:904
 msgctxt "input-name"
 msgid "UI Display Comment"
-msgstr ""
+msgstr "Benutzeroberfläche Kommentar anzeigen"
 
 #: src/emu/inpttype.ipp:905
 msgctxt "input-name"
 msgid "UI Clear"
-msgstr ""
+msgstr "Benutzeroberfläche Rücksetzen"
 
 #: src/emu/inpttype.ipp:906
 msgctxt "input-name"
 msgid "UI Zoom In"
-msgstr ""
+msgstr "Benutzeroberfläche Hineinzoomen"
 
 #: src/emu/inpttype.ipp:907
 msgctxt "input-name"
 msgid "UI Zoom Out"
-msgstr ""
+msgstr "Benutzeroberfläche Herauszoomen"
 
 #: src/emu/inpttype.ipp:908
 msgctxt "input-name"
 msgid "UI Default Zoom"
-msgstr ""
+msgstr "Benutzeroberfläche Standard Zoom"
 
 #: src/emu/inpttype.ipp:909
 msgctxt "input-name"
 msgid "UI Previous Group"
-msgstr ""
+msgstr "Benutzeroberfläche Vorherige Gruppe"
 
 #: src/emu/inpttype.ipp:910
 msgctxt "input-name"
 msgid "UI Next Group"
-msgstr ""
+msgstr "Benutzeroberfläche Nächste Gruppe"
 
 #: src/emu/inpttype.ipp:911
 msgctxt "input-name"
 msgid "UI Rotate"
-msgstr ""
+msgstr "Benutzeroberfläche Drehen"
 
 #: src/emu/inpttype.ipp:912
 msgctxt "input-name"
 msgid "Show Profiler"
-msgstr ""
+msgstr "Profiler anzeigen"
 
 #: src/emu/inpttype.ipp:913
 msgctxt "input-name"
 msgid "UI Toggle"
-msgstr ""
+msgstr "Benutzeroberfläche umschalten"
 
 #: src/emu/inpttype.ipp:914
 msgctxt "input-name"
 msgid "UI Release Pointer"
-msgstr ""
+msgstr "Benutzeroberfläche Zeiger Lösen"
```

Comment:
"Zeiger lösen" (lower case) or (IMHO better): "Zeiger freigeben"

Reply:
I like the seocndone better, changed.